docs/reference/Errors-and-Warnings.md

```diff
@@ -26,7 +26,7 @@ NuGet supports the following configuration properties.
 ## Errors
 
 | Group | Error Numbers |
-| --- | --- |
+|-------|---------------|
```

Comment:
I don't know why this is causing the docs bot to generate a warning, but it is warning, so please revert this change.

Reply:
This was actually my attempt at fixing it. I'll revert the change. No clue why the warning is showing up.